website/content/docs/integrations/consul/index.mdx

```diff
@@ -171,11 +171,11 @@ Consul, with some exceptions.
 
 * Nomad is not compatible with Consul Data Plane.
 
-|                   | Consul 1.16.0+ | Consul 1.17.0+ | Consul 1.18.0+ |
+|                   | Consul 1.17.0+ | Consul 1.18.0+ | Consul 1.19.0+ |
 |-------------------|----------------|----------------|----------------|
-| Nomad 1.7.0+      | ✅             | ✅             | ✅             |
-| Nomad 1.6.0+      | ✅             | ✅             | ✅             |
-| Nomad 1.5.0+      | ✅             | ✅             | ✅             |
+| Nomad 1.8.0+      | ✅             | ✅             | ✅            |
+| Nomad 1.7.0+      | ✅             | ✅             | ✅            |
+| Nomad 1.6.0+      | ✅             | ✅             | ✅            |
```

Comment:
@david-yu can you provide a link to the PR/Actions run where the `test-consul` tests successfully ran with the new Consul release?

Reply:
@tgross Thanks will check with the Consul team.

Reply:
From our call: Nomad owns the tests that give us the attestations for this matrix. We've got a green test for 1.19 here: https://github.com/hashicorp/nomad/actions/runs/9496700789/job/26171682960?pr=23308 so we can go ahead with this.